### ISSUE TITLE
Retracing vmcores in Podman fails

### ISSUE THREAD

**Issue by mgrabovsky** [abrt/retrace-server#423]
Interactively retracing vmcores in Podman fails with the message

    crash: /cores/retrace/repos/kernel/x86_64/usr/lib/debug/lib/modules/4.18.0-80.1.2.el8_0.x86_64/vmlinux: No such file or directory

    Usage:

      crash [OPTION]... NAMELIST MEMORY-IMAGE[@ADDRESS]	(dumpfile form)
      crash [OPTION]... [NAMELIST]             		(live system form)

    Enter "crash -h" for details.

Reported by @DaveWysochanskiRH.

**Comment by DaveWysochanskiRH** [comment]
There are a number of issues.  I tried this again and ran into other issues, I didn't even get to the above error with vmlinux yet.
1. Need rootless podman.  This seems to be somewhat a mess but I think this works with latest upstream.  However there's a few weird bugs like this bug in certain versions of RHEL8 where local users cannot run rootless podman if 
2. Copying vmcore into container is a problem due to size of vmcores (often 100GB or more).
3. Need both local users and ldap users to be able to use podman rootless.  There's some issues with this depending on which version but I think this is fixed in upstream (see https://bugzilla.redhat.com/show_bug.cgi?id=2092629 and related bugs such as https://bugzilla.redhat.com/show_bug.cgi?id=2063750 and  https://bugzilla.redhat.com/show_bug.cgi?id=2068088)
4. Above vmlinux error.  I think this may just need a copy the debuginfo files into the container but I'm not sure.

**Comment by DaveWysochanskiRH** [comment]
>  2. Copying vmcore into container is a problem due to size of vmcores (often 100GB or more).
I don't think we need to do this but we can use "-v" to bind mount in the vmcore and vmlinux files and any needed paths.

**Comment by DaveWysochanskiRH** [comment]
> > 5. Issues with 'AuthGroup' where tasks would fail with the following error in the retrace_log
> >    [2022-10-11 10:50:01] [E] Task failed: Unable to build podman container: time="2022-10-11T10:50:01-04:00" level=error msg="running `/usr/bin/newuidmap 1155992 0 174 1 1 231072 65536`: newuidmap: Target process 123456 is owned by a different user: uid:111 pw_uid:111 st_uid:111, gid:5555 pw_gid:111 st_gid:5555\n"
> 
> I had AuthGroup set in /etc/retrace-server/retrace-server.conf so that is why I got the above error. I needed to update the 'retrace' group in /etc/password as follows and this fixed the above. I wonder if that should be a standard procedure for installs when AuthGroup is used?
> 
> ```
> # usermod -g my-auth-group retrace
> # systemctl restart httpd
> ```

After the above I'm getting the following error:
~~~
[2022-10-17 13:11:52] [E] Task failed: Unable to build podman container: Error: failed to mount overlay for metacopy check with "" options: permission denied
~~~
